README.md

```diff
@@ -1,6 +1,6 @@
 # basic-setup
 
-Basic box setup, it's fairly customizable using bash/powershell and will work on gui-based and headless systems. Open a pr if you want something added.
```

Comment:
we may want to add a tag before all of this so that when we add back in windows support we have what was here before

Reply:
https://github.com/mrlunchbox777/basic-setup/releases/tag/powershell-exists